File: src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/OffenderRiskProfilerApplication.java
package uk.gov.justice.digital.hmpps.riskprofiler;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;

@SpringBootApplication
@EnableResourceServer
public class OffenderRiskProfilerApplication {

    public static void main(String[] args) {
        SpringApplication.run(OffenderRiskProfilerApplication.class, args);
    }

}

File: src/main/kotlin/uk/gov/justice/digital/hmpps/riskprofiler/OffenderRiskProfilerApplication.kt
package uk.gov.justice.digital.hmpps.riskprofiler

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
class OffenderRiskProfilerApplication {

  companion object {
    @JvmStatic
    fun main(args: Array<String>) {
      SpringApplication.run(OffenderRiskProfilerApplication::class.java, *args)
    }
  }
}
